### ISSUE TITLE
Chrome autofill appearing for autocomplete in 3.6.0

### ISSUE THREAD

**Issue by blackbaud-GavinNicol** [blackbaud/skyux-lookup#104]
### Expected behavior

Chrome autofill should not appear when clicking on the autocomplete component.

### Actual behavior

After upgrading to 3.6.0 of `skyux/lookup` the autofill window has started appearing when focusing on the autocomplete component

### Steps to reproduce

Stackblitz: https://stackblitz.com/edit/angular-r1mhlc

1. Type "red" into the search box
2. Select "Red"
3. Hit submit
4. Clear out the component
5. Lose and regain focus on the component
6. Notice the autofill dropdown.
7. Clicking the autofill dropdown results in the following error:
```
TypeError: Cannot read property 'toLowerCase' of undefined
```

**Comment by Blackbaud-SteveBrush** [comment]
Related: https://github.com/blackbaud/skyux-lookup/issues/93

**Comment by Blackbaud-BrianBlanchard** [comment]
This is still an issue on the latest Chrome (84).  `autocomplete="off"` works on Chrome, can you conditionally change that attribute value based on the browser?

**Comment by Blackbaud-AlexKingman** [comment]
@Blackbaud-BrianBlanchard Yep, we're already working on a fix for this that is intended to work across browsers. It will most likely be a directive that you can apply to any input element. We should have an update soon.

**Comment by Blackbaud-BrianBlanchard** [comment]
@Blackbaud-AlexKingman Great!  Just to confirm - this will be fixed in the v3 version of the library in addition to v4?  While we plan on moving to sky4 in the future, our current production issue is for the sky3 library.

**Comment by Blackbaud-AlexKingman** [comment]
@Blackbaud-BrianBlanchard [We've got a PR to fix v4](https://github.com/blackbaud/skyux-forms/issues/157), and will work on porting this backwards into v3 as soon as that's released.

**Comment by Blackbaud-KrisMahon** [comment]
Any updates on this?  How close are we to getting a SKYUX 4 fix?  We are seeing hundreds of failures from clients trying to use the autocomplete component and choosing an option from Chrome's autocomplete dropdown which then kills our SPA.
